components/eam/docs/tech-guide/homme.md

```diff
@@ -0,0 +1,26 @@
+# High-Order Methods Modeling Environment
+
+## Overview
+
+EAM using the a dynamical core (dycore)  from the High Order Method Modeling Environment (HOMME).  The EAM dycore solves the atmospheric primitive equations governing the evolution of velocity, density, pressure and temperature, as well as the transport of water species and related hydrometers, aerosols and other atmospheric constituents.  The governing equations are written in a vertically lagrangian terrain following mass coordinate.  They are discretized with second order finite differences in the radial direction and spectral finite elements in the horizontal (surface of the sphere) directions, and advanced in time with a 3rd order accurate 5 stage Runge-Kutta method.   Dissipation is added through the use of monotoncity contraints on some advectiion terms, explicitly added hyperviscosity, and a Laplacian-based sponge layer in the first few layers at the model top.  The transported species  makes use of an efficient interpolatory semi-Lagrangian method.   EAMv3 uses 80 layers in the vertical.   The use of the spectral finite element method allows EAMv3 to run on fully unstructured grids, including the cubed-sphere grid ([SE Atmosphere Grid Overview (EAM & CAM)](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/34113147)) which provides quasi-uniform resolution over the globe, and regionally refined meshes (RRM) which enhance horizontal resolution in regions of interest ([Library of Regionally-Refined Model (RRM) Grids](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/3690397775)).
+
+### References
+
```

Comment:
Don't break out the references separately.  Keep them inline as done in other sections like P3 and MAM.  The bib function makes a better version of this.

Reply:
I've added the references in the overview text as suggested.